config/witnesses-prod.yaml

```diff
@@ -0,0 +1,8 @@
+Witnesses:
+  - DEV:ArmoredWitness-quiet-hill+36ccdbc6+AYla/cX7GRGOIBg9nM9PFZANcMLAR2XLR0nD9V8siErf
```

Comment:
These look like dev witnesses, I guess they're just being grandfathered in.
Just wondering if we want to clear them out since we'll have CI/wave0 devices ready shortly?

Reply:
This file can't be empty or the deployment breaks :-)

The plan will be to wipe prod before real launch anyway, so I decided to just leave this as is. It's already been running with the built-in list which had everything in there, so this is not making anything worse.